src/pages/pass-finalisation/cat-b/pass-finalisation.module.cat-b.page.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
```

Comment:
This page should follow this file name pattern

waiting-room.cat-b.page.module